[SG-796] Mobile getting started updates (#29)
bitwarden/contributing-docs: docs/getting-started/server/tunnel.md

1. Navigate to <https://ngrok.com/> and sign up for an account

> :::note
> 
> Install ngrok via [brew](https://formulae.brew.sh/cask/ngrok) to avoid the limitation of one instance per account.
> 
> :::
> 
> 2. Follow the [official instructions](https://dashboard.ngrok.com/get-started/setup) to download.
>    Can also be installed via . 

Reviewer: The note is displayed outside the list. Thoughts on moving it into `2`?

```suggestion
2. Follow the [official instructions](https://dashboard.ngrok.com/get-started/setup) to download. Or
   install using [brew](https://formulae.brew.sh/cask/ngrok) which supports multiple instances per
   account.
```
